lithops/standalone/master.py

```diff
@@ -47,7 +50,9 @@
 REUSE_WORK_QUEUE_NAME = 'all'
 
 exec_mode = 'consume'
-workers = {}
+mp_manager = mp.Manager()
+workers = {} # TODO: consider change it to mp_manager.dict()
+workers_state = mp_manager.dict()
```

Comment:
Few weeks ago I checked the performance of the master service (cpu and mem consumption) and I saw that it is a very lightweight service, so I deleted all the functions that where running with mp.Process(), and switched all them to Threads(). So now the master service is a single process using Threads. Is there a particular reason that you need the multiprocessing library and the Manager() back here?

Reply:
I wonder if it is time to install a redis in the master VM and keep both work-queues and workers+state there.

Currently, everything is stored in the master service using dicts and lists. If for some reason the master service crashes, it is automatically reloaded, but all of this data is lost: This means all the pending task in the work queues are lost, and the list of available workers is also lost, so workers must be deleted and then we have to create new ones. Maybe, to improve Lithops resilience, we can do the same as Ray do with the head node and store all this data in the redis. This way, if master service crashes due an unhandled exception, it can reload all this data.

Reply:
@JosepSampe Removed multiprocessing.
Regarding `Redis` - good idea! I think it is a time already.